build.gradle

```diff
@@ -70,6 +70,32 @@ dependencies {
     implementation 'com.github.luben:zstd-jni:1.5.5-4'
     implementation 'org.lz4:lz4-java:1.8.0'
     implementation 'org.xerial.snappy:snappy-java:1.1.10.5'
+    implementation 'software.amazon.msk:aws-msk-iam-auth:1.1.4'
```

Comment:
Can we use more modern versions of these libraries? These are somewhat out of date

Reply:
Tested with commit 4c37ca950c911ee87d837b51115127e5d6afdb21 and was able to connect. 
So @robbavey please could you review it again 🙏

Reply:
In 4c37ca950c911ee87d837b51115127e5d6afdb21 you can see the difference switching from `1.1.4` to `2.2.0`. While there is a consistent part of AWS libraries (part of groups `software.amazon.awssdk` and `com.amazonaws`) with this new version it adds:
- `commons-codec:commons-codec`
- `commons-logging:commons-logging`
- `io.netty:netty-*`
- `org.reactivestreams:reactive-streams`
- `org.apache.httpcomponents` `httpclient` and `httpcore`

While Netty http and http2 codecs are present it was added also a dependency on Apache's `httpclient` so seems redundant.
The risky one is Netty, because used in other Logstash plugins.